File: src/main/java/me/nobaboy/nobaaddons/commands/subcommands/RefillPearlsCommand.java
package me.nobaboy.nobaaddons.commands.subcommands;

import me.nobaboy.nobaaddons.NobaAddons;
import me.nobaboy.nobaaddons.commands.ISubCommand;
import me.nobaboy.nobaaddons.features.dungeons.PearlRefill;

public class RefillPearlsCommand implements ISubCommand {
	@Override
	public String getName() {
		return "refillpearls";
	}

	@Override
	public boolean isEnabled() {
		return NobaAddons.config.refillPearls;
	}

	@Override
	public void run(String[] args) {
		PearlRefill.refillPearls();
	}
}

File: src/main/kotlin/me/nobaboy/nobaaddons/commands/subcommands/RefillPearlsCommand.kt
package me.nobaboy.nobaaddons.commands.subcommands

import me.nobaboy.nobaaddons.NobaAddons
import me.nobaboy.nobaaddons.commands.ISubCommand
import me.nobaboy.nobaaddons.features.misc.PearlRefill

class RefillPearlsCommand : ISubCommand {
    override val name: String = "refillPearls"

    override val isEnabled: Boolean
        get() = NobaAddons.config.dungeons.refillPearls

    override fun run(args: Array<String>) {
        PearlRefill.refillPearls(true)
    }
}